### ISSUE TITLE
Which is more important to this group?

### ISSUE THREAD

**Issue by werdnanoslen** [open-austin/project-ideas#10]
I research civic media and how civic hacking groups prioritize things, and I'm curious about whether Open Austin prioritizes promotion of ideas and experiments over targeted problem solving. This distinction is important to overall organization in whether contributors should spread out and try new things or focus on one problem (and its set of issues) at a time. Looking through the website, it seems the group espouses some high-level goals that befits a community, but I'm interested in how the people currently making up this organization operate on projects at the moment. This might be too meta for this medium, but it's I feel this is in-line with coming up with and managing project ideas.

tl;dr: how often does everyone come together to focus on a problem or core set of issues, or is this a more umbrella-like organization for separate projects?

**Comment by luqmaan** [comment]
I'm not sure if I'm the best person to answer this, but so far in my experience we don't plan to focus on particular types of issues.

People just work on things they find interesting. Sometimes a project is interesting because it solves a problem you're affected by. Other times it is interesting because of the technologies used. Or the people working on the project.

So, any idea goes. Do you have any in mind?

**Comment by werdnanoslen** [comment]
I think you may be as good as anyone else to say so, unless there's a sort of cultural leader in the group? I'm interested to hear others' opinions as well, though.

No specific ideas for Austin right now, but a lot of ideas I've seen here and in (Code for) Atlanta involve I/O with maps. So I've started working on a [boilerplate for that](https://github.com/werdnanoslen/mobile-map-io).

**Comment by mateoclarke** [comment]
Yeah, I'd agree with @luqmaan about the two drivers for involvement. Either personal interest because the domain (policy issue) or technology (stack or problem) resonates with the volunteer.

Also to answer your question in a different way, our Hack Team definitely work more as an umbrella organization to tackle various issues. We have small teams of volunteers working on projects at the same time. Rarely do we all focus on one sole issue.

We have 2-3 big events per year when we get as many people together to contribute their ideas, but these ideas usually diverge into different projects, not one problem and its set of issues.

Interesting suggestion though. I haven't heard of that model for any of the Code for America affiliated brigades. Have you?

**Comment by luqmaan** [comment]
We also do a lot of map apps.

While you're thinking about a good boilerplate for maps, I like the way we do it in https://github.com/open-austin/austingreenmap.

- react
- react-leaflet
- scss
- gulp
- hosted on github pages (see the gh-pages branch)
- soon will be publishable as native/cordova ios/android apps https://github.com/open-austin/austingreenmap-cordova

**Comment by luqmaan** [comment]
Closing this issue, as @werdnanoslen's question seems to be answered. Feel free to reopen.
